requirements.txt

```diff
@@ -2,15 +2,22 @@ Babel==1.3
 Django==1.8.2
 Jinja2==2.7.3
 MarkupSafe==0.23
+PyJWT==1.3.0
 Pygments==2.0.2
 alabaster==0.7.4
-argparse==1.2.1
 dj-database-url==0.3.0
 django-extensions==1.5.5
 docutils==0.12
+fitbit==0.1.3
+gunicorn==19.3.0
+oauthlib==0.7.2
 psycopg2==2.6
+python-dateutil==2.4.2
```

Comment:
You know, I'm not sure we're actually using this.  I think the rest of the new imports are ok (most come from python social and and it's dependencies).